service/device/registry/internal/src/test/java/org/eclipse/kapua/service/device/registry/internal/DeviceRegistryServiceTest.java

```diff
@@ -31,6 +32,7 @@
 import static org.eclipse.kapua.commons.security.KapuaSecurityUtils.doPriviledge;
 import static org.eclipse.kapua.service.device.registry.DeviceCredentialsMode.LOOSE;
 
+@Ignore
```

Comment:
Hi @andrejn-ct ,

I forgot that if we are running Cucumber tests, jUnit tests must be ignored, otherwise they fails.

Regards, 

_Alberto_